pkg/datapath/linux/probes/probes.go

```diff
@@ -134,8 +134,6 @@ func NewProbeManager() *ProbeManager {
 		var features Features
 		out, err := exec.WithTimeout(
 			defaults.ExecTimeout, "bpftool", "-j", "feature",
-			"probe", "filter_out",
```

Comment:
Hmm maybe keep `probe`? It's more explicit if other subcommands are added to `feature` one day.